decentraland/unity-explorer
# feat: authentication ui flow

Introduces an initial authentication UI flow as described here: https://www.notion.so/decentraland/EA-Authentication-2b04be4ce3534852a7ac8c4376cbdd64?pvs=4

## Test steps
1. Select a realm from the selector. `peer.decentraland.org` for real profiles
2. Scroll down the debug menu and click: `WEB3 AUTHENTICATION: Open Auth UI`
3. Follow the UI flow
4. The signing transaction should work as expected
5. In the final step, check that your profile name is displayed correctly
